package.json

```diff
@@ -112,6 +114,9 @@
     "webpack-bundle-analyzer": "4.4.2",
     "webpack-cli": "3.3.11"
   },
+  "tap": {
```

Comment:
@travis-payne Same question here as above regarding the skipping of the coverage checks.